plugins/Jobs.cpp

```diff
@@ -1195,6 +1196,34 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
         return true;
     }
 
+    // Requeue a job for which a getJob(s) command could not complete.
+    else if (SIEquals(requestVerb, "RequeueJobs")) {
```

Comment:
By doing this, you are exposing this new command, right? Should we try to mark this as internal somehow or not worth it?

Reply:
It's only exposed to the same places that can make new jobs anyway, I don't see how there's any greater risk here.